### ISSUE TITLE
Trigger filter selection with hash url

### ISSUE THREAD

**Issue by quentin-th** [Robbertdk/wordpress-ajax-filter-posts#34]
Hi, it's me again :)

I wanted to use url to select a filter when opening the page with the hash. 
Ex: when accessing to https://my-website.com/#filter=apple, the ajax filter for "apple" would be automatically applied.

I almost managed to do it, except that, after loading the page through the url hash, when I clicked from one filter to another the ajax requests were duplicated, so the results were not good, I couldn't figured out why.

I don't have time to look into it anymore but I think it could be a good feature !

**Comment by Robbertdk** [comment]
@quentin-th Good idea! I've implemented this functionality for clients outside of WordPress and I can integrate that code with this package. Nice benefit of this functionality is that people can share urls with the filter applied and directly see the filtered posts.

Notice that, although this package skips the need of having urls for categories that list posts specific to that category, those pages still exist. This is a default WordPress functionality. So when you create a category named cars for your posts. You still have the url  www.mysite.com/category/cars/, depending on your url structure.  You can also alter these pages [by setting a custom category template](https://codex.wordpress.org/Category_Templates).
